CONTRIBUTING.md

```diff
@@ -16,8 +16,10 @@ as a contributor, have the legal right to make the contribution.
 
 See the [DCO](DCO) file for the full text of what you must agree to.
 
-To signify that you agree to the DCO for a commit, you add a line to the git
-commit message:
+### Option 1: commit message signoffs
```

Comment:
markdown lint is complaining that these headers are inconsistent with the rest of the doc

```suggestion
### Option 1: commit message signoffs ###
```